index.html

```diff
@@ -173,126 +173,404 @@ <h2>
+                <ol>
+                  <li>
+                    If <a>options</a> are equivalent to the current <a>pointer-lock options</a> then the <a>pointer-lock target</a>
+                    must be updated to this element, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
+                  </li>
+                  <li>
+                    If <a>options</a> are not equivalent to the user agent's current
+                    <a>pointerLockOptions</a> and <a>options</a> are not supported:
+                    <ol>
+                      1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                      Queue an element task</a> on the 
+                      <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                      User Interaction task source</a>, given this, to perform the following steps:
+                      <ol type='i'>
+                        <li>
+                          <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                          <a>pointerlockerror</a> at <a>this</a>'s 
+                          <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                        </li>
+                        <li>
+                          <a>Reject</a> the promise with a "{{NotSupportedError}}" {{DOMException}}.
+                        </li>
+                      </ol>
+                    </ol>
+                  </li>
+                  <li>
+                    If <a>options</a> are not equivalent to the current <a>pointer-lock options</a> and 
+                    <a>options</a> are supported then perform the following steps
+                    <ol type='i'>
+                      <li>
+                        The user agent's <a>pointer-lock target</a> must be updated to this element
+                      </li>
+                      <li>
+                        The user agent's <a>pointer-lock target</a> must be updated to this element
+                      </li>
+                      <li>
+                        The current <a>pointer-lock options</a> must be updated to <a>options</a>
+                      </li>
+                      <li>
+                        a <a>pointerlockchange</a> event must be sent
+                      </li>
+                      <li>
+                        the promise must <a>resolve</a>
+                      </li>
+                    </ol>
+                  </li>
+                </ol>
+              </li>
+              <li>
+                If any element (including this one), whose <a>shadow-including
+                root</a> is same as this element's <a>shadow-including root</a> and, is
+                already locked (or pending lock) then perform the following steps
```

Comment:
I think this step is better phrased as something like

> If the user agent's current pointer-lock target's shadow-including root is the same as this element's shadow-including root, then:

Reply:
Done.